ui/app/components/ui/button/button.component.js

```diff
@@ -37,6 +37,7 @@ const Button = ({ type, submit, large, children, rounded, className, ...buttonPr
     )}
     { ...buttonProps }
   >
+    {icon && <span className="button__icon">{icon}</span>}
```

Comment:
It would be nice to have an additional story that utilized this icon

Reply:
Oh, I meant a story in `button.stories.js` that utilized this new prop! The addition to `icon.stories.js` is good too though